src/Symfony/Component/Validator/Constraints/RangeValidator.php

```diff
@@ -11,15 +11,26 @@
 
 namespace Symfony\Component\Validator\Constraints;
 
+use Symfony\Component\PropertyAccess\Exception\NoSuchPropertyException;
+use Symfony\Component\PropertyAccess\PropertyAccess;
+use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
 use Symfony\Component\Validator\Constraint;
 use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\ConstraintDefinitionException;
 use Symfony\Component\Validator\Exception\UnexpectedTypeException;
 
 /**
  * @author Bernhard Schussek <bschussek@gmail.com>
  */
 class RangeValidator extends ConstraintValidator
 {
+    private $propertyAccessor;
+
+    public function __construct(?PropertyAccessorInterface $propertyAccessor = null)
```

Comment:
We don't use nullable typehints when the default `null` value is provided as it's considered redundant.